single-contacts.php

```diff
@@ -668,14 +669,14 @@ class="dt_multi_select <?php echo esc_html( $class ) ?> select-button button ">
     </div>
 
     <div class="reveal" id="make_user_from_contact" data-reveal data-reset-on-close>
-
-        <p class="lead"><?php esc_html_e( 'Create User', 'disciple_tools' )?></p>
+        <h1><?php echo esc_html_x( 'Make User From Contact', 'Make user modal', 'disciple_tools' )?></h1>
 
         <?php if ( isset( $contact['corresponds_to_user'] ) ) : ?>
-            <p><?php esc_html_e( "This contact already represents a user", 'disciple_tools' ) ?></p>
+            <p><strong><?php echo esc_html_x( "This contact is already connected to a user.", 'Make user modal', 'disciple_tools' ) ?></strong></p>
         <?php else : ?>
 
-        <p><?php esc_html_e( "This will invite this contact to Disciple.Tools as a multiplier", 'disciple_tools' ) ?></p>
+        <p><?php echo esc_html_x( "This will invite this contact to become as a of this system. By default, they will be given the role of a 'multiplier'.", 'Make user modal', 'disciple_tools' ) ?></p>
```

Comment:
This will invite this contact to become as a of this system.
Missing a word?

Reply:
This will invite this contact to become a user of this system

Reply:
@mikeallbutt did you change this?

Reply:
yes you did.